bgpd/bgp_mplsvpn.c

```diff
@@ -760,6 +843,13 @@ void ensure_vrf_tovpn_sid_per_af(struct bgp *bgp_vpn, struct bgp *bgp_vrf,
 	if (!bgp_vpn || !bgp_vpn->srv6_locator)
 		return;
 
+	if (bgp_vrf->vrf_id == VRF_UNKNOWN) {
+		if (debug)
+			zlog_debug("%s: vrf %s: vrf_id not set, can't set zebra vrf SRv6 SID",
+				   __func__, bgp_vrf->name_pretty);
+		return;
+	}
+
```

Comment:
Could you describe an example when we might be in this case?

Reply:
In some cases, `vrf_id` can be `VRF_UNKNOWN`. This can happen, for example, when you configure a BGP instance for a VRF, but the VRF does not yet exist in the data plane.

Reply:
thanks